aten/src/ATen/QTensorImpl.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
```

Comment:
Shall we move it to ATen/quantized? (not ATen/native/quantized). It's similar to specific code in ATen/cuda for example.

Reply:
Reminder to move it :)

Reply:
@dzhulgakov please review, I also added intrusive_ptr and intrusive_from_this